### DISCUSSION TITLE
How to create a background image that fits the screen.

### DISCUSSION THREAD

**Discussion by RylanYancey** [emilk/egui#3383] [Q&A]
Currently, I'm rendering an image like this:         
```rs
egui::CentralPanel::default().show(&gui.platform.context(), |ui| {
            ui.image(splash, gui.platform.context().screen_rect().size());
        });
```

However, this isnt perfect. Heres what this creates: 

![image](https://github.com/emilk/egui/assets/76256852/5113c70a-f826-4049-bc20-ed2e3dc67af2)

The image doesnt fit the screen perfectly. 

On top of that if I add the following code, the background image doesnt even show anymore! Just the button.

```rs
        egui::CentralPanel::default().show(&gui.context(), |ui| {
            if ui.button("Click me!").clicked() {
                println!("You clicked the button! :D");
            }
        });
```

What is the proper way to make a background image?

Thanks!

**Comment by JumpyLionnn** [comment]
The problem is that `ui.image()` allocates space and this space is shrunk by the `CentralPanel` margin.

You can try to use this at the top of your ui code:
```rust
egui::Image::new(egui::include_image!("../assets/icon.png")).paint_at(ui, ui.ctx().screen_rect());
```
